packages/chakra-ui/src/range-field/RangeField.stories.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { useCallback } from "react";
+import { RangeField } from "./RangeField";
+import { fieldAtom, formAtom, useForm } from "form-atoms";
+import { StoryForm } from "../stories";
+import { zodValidate } from "form-atoms/zod";
+import { z } from "zod";
+import { NumberFieldAtom } from "@react-last-field/field/src";
```

Comment:
```suggestion
import { NumberFieldAtom } from "@form-atoms/field/src";
```

meanwhile the project has been renamed.

You can sync your fork in github, and then rebase this branch on main!

Reply:
[7d1c68d](https://github.com/MiroslavPetrik/form-atoms-field/pull/8/commits/7d1c68d1cf89dfe132debdf2960c4030647ff626)